src/shell/main.js

```diff
@@ -109,6 +110,16 @@ async function START() {
         const focusedWindow = BrowserWindow.getFocusedWindow()
         if (focusedWindow) focusedWindow.webContents.closeDevTools()
     })
+    ipcMain.on('copy-image-to-clipboard', (event, arg) => {
+        try {
+            // Always use 'image/png' for now as it seems like nothing supports anything but image/png
+            clipboard.writeBuffer('image/png', arg.buffer);
```

Comment:
So, does this work when arg.buffer contains webp or other media type ? is this `image/png` just like the index key every app look into to find the image buffer ?